user-registration-app/src/app/views/live-view/live-view.component.html

```diff
@@ -305,279 +303,427 @@
                 <span class="card-title activator text-darken-4" style="font-weight:bold;">
-                  Clean Coding Practices</span>
-                <div style="font-weight:bold;">Sushrut Shringarputale</div>
-                <div style="font-weight:bold;">Location: Auditorium (Workshop Room 4)</div>
-                <div style="font-weight:bold;">Time: 9:00 AM - 9:45 AM Sunday</div>
+                  Simplifying Python Code with Enhanced Functional Programming</span>
+                <div style="font-weight:bold;">Brian Ouzomgi & Ian D’Ambrosio</div>
+                <div style="font-weight:bold;">Location: 126 Business Building (Workshop Room 3)</div>
+                <div style="font-weight:bold;">Time: 9:00 PM - 9:45 PM Saturday</div>
               </div>
               <div class="card-reveal">
-                <span class="card-title">Clean Coding Practices<i
+                <span class="card-title">Simplifying Python Code with Enhanced Functional Programming<i
                   class="material-icons right">close</i></span>
-                <p>Could a stranger read and understand the code you write?
-                  Come learn how to write clean, readable, and industry-standard code.
+                <p>In this workshop, you will learn user-centered design strategies
+                  for creating a product that is intuitive and easy to use. You will
+                  come away with an understanding of established design practices as
+                  well as some tips and tricks to help guide the design process!
                 </p>
+                <p class="card-requirements">This workshop has software requirements:</p>
+                <ul>
+                  <li class="wrksp-req">Python Interpreter</li>
+                  <li class="wrksp-req">Text Editor of your choice</li>
+                </ul>
               </div>
             </div>
           </div>
         </div>
       </div>
 
-      <div class="col-md-6 col-lg-3 nopadding">
+      <div class="col-md-6 col-lg-4 nopadding">
         <span class="workshop-track">ADVANCED</span>
         <div class="row">
           <div class="container-fluid py-4">
             <div class="col-12 p-md-0 px-4">
               <div class="card advanced">
-                <div class="card-image waves-effect waves-block waves-light">
-                  <img class="activator" src="../../../assets/images/royce.png">
-                  <a class="btn halfway-fab waves-effect waves-light red activator">click to learn more</a>
+                <div class="card-image">
+                  <img class="activator" src="../../../assets/images/workshop_leaders/RoyceD'Souza.jpg">
+                  <a class="waves-effect waves-light activator red btn">click to learn more</a>
                 </div>
                 <div class="card-content pt-5">
                   <span class="card-title activator" style="font-weight:bold; color: white;">Blockchain with Blockchain@PSU</span>
```

Comment:
The location and time should be Auditorium (Workshop Room 4) and 6:00 PM - 6:45 PM

Reply:
It's not visible in the commit because it wasn't something that was changed

Reply:
I can't believe you're actually checking that I didnt think you would lmao

Reply:
Fixed

Reply:
I think you underestimate how much I don't want to do EE